index.js

```diff
@@ -1,14 +1,20 @@
 const mongoose = require('mongoose');
 
+
 // Import of the model Recipe from './models/Recipe.model.js'
 const Recipe = require('./models/Recipe.model');
 // Import of the data from './data.json'
 const data = require('./data');
+const { findOneAndUpdate } = require('./models/Recipe.model');
```

Comment:
This variable is not doing nothing here since you are not using it. You have access to `.findOneAndUpdate()` method and there's no need to initialize this variable right here.